python_anvil/api.py

```diff
@@ -396,7 +392,23 @@ def create_etch_packet(
             )
 
         payload = mutation.create_payload()
-        variables = payload.dict(by_alias=True, exclude_none=True)
+
+        # FIXME: do something here to prevent or fix pydantic from converting
+        # files into "SerializationIterator" objects
+        variables = payload.model_dump(by_alias=True, exclude_none=True)
+
+        # print('Files', variables["files"])
```

Comment:
need this?

Reply:
good eye -- looks like debug code @fostimus, I pulled it out and retested in my PR https://github.com/anvilco/python-anvil/pull/69